README.md

```diff
@@ -533,3 +534,7 @@ To add a resource or contribute, please check out our contribution guide [here](
 ### Standard Template Library
 
 - [C++ Standard Template Library in Practice](https://www.udemy.com/course/c-standard-template-library-in-practice/learn/)
+
+### Programming Courses Free Online
+
+- [Programming Courses Free Online](https://www.sololearn.com/learning)
```

Comment:
This should be the title of the resource you're providing, so in this case the title should be "SoloLearn"